src/pykeen/models/unimodal/rgcn.py

```diff
@@ -24,9 +24,55 @@
 class RGCN(
     ERModel[torch.FloatTensor, RelationRepresentation, torch.FloatTensor],
 ):
-    """An implementation of R-GCN from [schlichtkrull2018]_.
+    r"""An implementation of R-GCN from [schlichtkrull2018]_.
 
-    This model uses graph convolutions with relation-specific weights.
+    The Relational Graph Convolutional Network (R-GCN) [schlichtkrull2018] builds on an adapted GCN that can be applied
+    on multi-relational data such as Knowledge Graphs.
+    More concretely, the RGCN model consists of two major parts: i.) a GCN-based entity-encoder that computes
```

Comment:
Maybe the whole part about the GCN encoder should go to the `RGCNRepresentations` class, and only be linked here?

Reply:
Agreed!

Reply:
Same could be said about the decompositions